docs/TRANSLATIONS.md

```diff
@@ -25,7 +25,7 @@ Main:
 Additional:
 - Text-to-speech strings for navigation: [`data/strings/sound.txt`](../data/strings/sound.txt)
 
-- Android stores description: [`android/src/google/play/listings/`](../android/src/google/play/listings/)
+- Android stores description: [`android/src/google/play/listings/`](../android/src/google/play/)
```

Comment:
```suggestion
- Android stores description: [`android/src/google/play/`](../android/src/google/play/)
```

Reply:
@biodranik  but there does not exist any folder named "listings" in  google/play/

Reply:
Exactly, so that's why I suggest to remove it.